api/core/v2/rbac.proto

```diff
@@ -68,28 +72,31 @@ message Subject {
 // ClusterRoleBinding grants the permissions defined in a ClusterRole referenced
 // to a user or a set of users
 message ClusterRoleBinding {
+  option (gogoproto.face) = true;
+  option (gogoproto.goproto_getters) = false;
+
   // Subjects holds references to the objects the ClusterRole applies to
   repeated Subject subjects = 1 [(gogoproto.jsontag) = "subjects", (gogoproto.nullable) = false];
 
   // RoleRef references a ClusterRole in the current namespace
   RoleRef role_ref = 2 [(gogoproto.jsontag) = "roleRef", (gogoproto.nullable) = false];
 
-  // Name of the ClusterRoleBinding
-  string name = 3 [(gogoproto.jsontag) = "name"];
+  // Metadata contains name, namespace, labels and annotations
+  ObjectMeta metadata = 4 [(gogoproto.embed) = true, (gogoproto.jsontag) = "metadata,omitempty", (gogoproto.nullable) = false];
 }
 
 // RoleBinding grants the permissions defined in a Role referenced to a user or
 // a set of users
 message RoleBinding {
+  option (gogoproto.face) = true;
+  option (gogoproto.goproto_getters) = false;
+
   // Subjects holds references to the objects the Role applies to
   repeated Subject subjects = 1 [(gogoproto.jsontag) = "subjects", (gogoproto.nullable) = false];
 
   // RoleRef references a Role in the current namespace
   RoleRef role_ref = 2 [(gogoproto.jsontag) = "roleRef", (gogoproto.nullable) = false];
```

Comment:
```suggestion
  RoleRef role_ref = 2 [(gogoproto.jsontag) = "role_ref", (gogoproto.nullable) = false];
```

Reply:
Not totally related.. but you fixed `resource_names` in this pr and this could save us some problems down the road.